scipy/stats/tests/test_levy_stable_fitstart.py

```diff
@@ -0,0 +1,91 @@
+# Test scipy.stats.levy_stable._fitstart().
+
+import numpy as np
+import pytest
+
+from scipy.stats import levy_stable
+from scipy.stats import rv_continuous as rvc
+
+class TestLevy_stable:
+    def test_neg_beta(self):
+        # Confirm that _fitstart() gets same estimate of alpha for +beta and -beta.
+        rvc.random_state = 2021
+        NPOINTS = 1001
+        ALPHA = 1.5
+        BETA_POS = 1
+        
+        draw = levy_stable.rvs(alpha=ALPHA, beta=BETA_POS, size=NPOINTS)
+        # Fit stable distribution to positive and negative mirror image data, examine alpha.
+        alpha_pos, alpha_neg = [levy_stable._fitstart(data)[0] for data in [draw, -draw]]
+        
+        assert np.isclose(alpha_pos, alpha_neg)
```

Comment:
Here and elsewhere, it is likely to be far more helpful to use `assert_allclose()` from the NumPy testing infrastructure than it is to use a plain `assert` on the boolean return value of the plain `isclose()` function, because `assert_allclose()` is specifically designed for testing purposes and clearly reporting the number and magnitude of differences.

Reply:
"far more helpful to use `assert_allclose()`"

Thanks.  That would be sound advice for sure in a context that could involve multiple values.  In `_fitstart()` the parameter lookups are assumed to be scalars so there is arguably less gain from the extra output of `assert_allclose()`, and perhaps potential for incorrectly implying that multiple values are potentially involved.  Thoughts?  Would you still advice using `assert_allclose()`?

Reply:
I've moved the tests to a better neighbourhood in test_distributions.py around line 3052 and rewritten them more in the style of what was there already.  The tests now mostly use numpy testing functions, except for a few plain asserts involving inequalities--is there a better way to do those?